tests/sqlparser_sqlite.rs

```diff
@@ -431,6 +431,35 @@ fn invalid_empty_list() {
     );
 }
 
+#[test]
+fn parse_start_transaction_with_modifier() {
+    let (supported_dialects, unsupported_dialects) =
+        partition_all_dialects_by_inclusion(vec!["generic", "sqlite"]);
```

Comment:
it isn't clear to me that using a String is preferable here to simply explicitly referring to `SqliteDialect` and `GenericDialect`

Using a string defers any errors to runtime rather than compile time

Reply:
Thanks for the feedback!

(I have not yet been able to learn Rust properly, so I may be missing the point..)
I first tried to pass the arguments like this
```rust
partition_all_dialects_by_inclusion(vec![Box::new(SQLiteDialect {}), Box::new(GenericDialect {})])
```
but I got the following error indicating that trait objects cannot be compared.
```
error[E0277]: can't compare `dyn dialect::Dialect` with `dyn dialect::Dialect`
    --> src/test_utils.rs:230:36
     |
230  |         .filter(|d| !dialect_names.contains(d))
     |                                    ^^^^^^^^ no implementation for `dyn dialect::Dialect == dyn dialect::Dialect`
     |
     = help: the trait `PartialEq` is not implemented for `dyn dialect::Dialect`
     = note: required for `Box<dyn dialect::Dialect>` to implement `PartialEq`
```
I couldn't think of a suitable way to deal with this, so I changed to using String which can be compared.
But yes, this is not a good solution, so you can skip it!